.github/workflows/pull-request-queue.yaml

```diff
@@ -14,9 +14,19 @@ jobs:
     name: enqueue
     runs-on: ubuntu-20.04
     steps:
+      - name: Query vault
+        id: vault
+        uses: hashicorp/vault-action@v2.0.1
+        with:
+          exportEnv: false
+          url: https://volt.cap.explore.suse.dev/
+          token: ${{ secrets.VAULT_TOKEN }}
+          secrets: |
+            secret/data/kubecf "github-access-token" | GH_BOT_TOKEN
```

Comment:
Hmm, does this work? I though we needed to use backticks for quoting here (because JSONata)?

Reply:
In some previous work with Vlad the issue with the dash/minus characters in the field name was fixed by double-quotes. so I used them here also.

Reply:
Seems to work in actual deployment, so 👍